framework/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set version = "5.1.20201125.2323" %}
-{% set git_rev = "f00b325940c0f7844d650607ff5fea52c23a4462" %}
+{% set git_rev = "cdd49013b63aa72cb3343ff14fd66d5f88b0182f" %}
```

Comment:
We are intending on modifying the conda scripts to take the `version` and `git_rev` as environment variables passed down through dependent builds. Having tons of commits to modify these two strings makes it harder to follow what is going on in the repo.

Reply:
TBH I think the way that things are done now is not good. The conda repository should be combined with the main mantid repository as I think conda is going to be the way forward - not just an experimental distribution mechanism. This allows much simpler usage in the `meta.yam` such as `version: {{ GIT_DESCRIBE_TAG }}` removes and the complex things that are done as part of `ENABLE_CPACK` in mantid as well as many of the python scripts required in the `conda_recipes` repository.

Reply:
> {% set git_rev = "cdd49013b63aa72cb3343ff14fd66d5f88b0182f" %}

As soon as the mantid changes are merged we can start using `master`

Reply:
This can now be safely overwritten by the update scripts. All necessary changes are in master.